Geometry/HGCalCommonData/interface/HGCalDDDConstants.h

```diff
@@ -29,14 +29,8 @@ class HGCalDDDConstants {
 
   std::pair<int,int>  assignCell(float x, float y, int lay, int subSec,
 				 bool reco) const;
-  std::pair<int,int>  assignCellSquare(float x, float y, float h, float bl, 
-				       float tl, float alpha, 
-				       float cellSize) const;
```

Comment:
@bsunanda - now that there is only hexagonal geometry left shouldn't "Hexagon" be dropped from the methods names?

Reply:
Yan - there will be new versions to be introduced soon. I would like to leave this as it is. New mode will be introduced next